deployment/build.windows.amd64

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+#= Windows 7+ amd64 (x64) .zip
+
+set -o errexit
+set -o xtrace
+
+# Version variables
+NSSM_VERSION="nssm-2.24-101-g897c7ad"
+NSSM_URL="http://files.evilt.win/nssm/${NSSM_VERSION}.zip"
+FFMPEG_VERSION="ffmpeg-4.2.1-win64-static"
+FFMPEG_URL="https://ffmpeg.zeranoe.com/builds/win64/static/${FFMPEG_VERSION}.zip"
```

Comment:
are you sure we're using those bits, not some other? I recall us making some jf-ffmpeg for Windows... cc @EraYaN

Reply:
This is unchanged from the existing build infra, so it can be altered separately if need-be.